linear_algebra/src/polynom_for_points.py

```diff
@@ -27,6 +27,12 @@ def points_to_polynomial(coordinates: list[list[int]]) -> str:
     f(x)=x^2*-1.0+x^1*-0.0+x^0*-2.0
     >>> print(points_to_polynomial([[1, 5], [2, 2], [3, 9]]))
     f(x)=x^2*5.0+x^1*-18.0+x^0*18.0
+    >>> print(points_to_polynomial([[1, 1], [1, 2], [1, 3]]))
+    x=1
+    >>> print(points_to_polynomial([[1, 1], [2, 2], [2, 2]]))
```

Comment:
Please try removing the `print()`...
```suggestion
    >>> points_to_polynomial([[1, 1], [1, 2], [1, 3]])
    x=1
    >>> points_to_polynomial([[1, 1], [2, 2], [2, 2]])
```

Reply:
> Please try removing the `print()`...

@cclauss now check I removed the print statement but I did't see any major change in output

Reply:
That sounds right because calling `print()` is not usually needed in doctests.  Can you please remove `print()` from all tests in this file then I think we can land this.

Reply:
Sure

Reply:
hi @cclauss check I removed the print from remaining place of polynom_for_points test cases